docs/guides/esp32/build_app_and_commission.md

```diff
@@ -53,9 +53,7 @@ functionality can still work fine.
 
 -   Matter
 
-    Before running any other build command, the scripts/activate.sh environment
-    setup script should be sourced at the top level. This script set up a Python
-    environment with libraries used to build and test.
+    Activate the Matter environment.
```

Comment:
I think this should be clearly documented as needing to be run after `export.sh`.

Reply:
This is placed exactly below the section which performs sourcing `esp-idf/export.sh`, still mentioned the same.

Reply:
Right, but nothing says "do these sections in order" (and in fact the ccache thing can be done in any order wrt the other bits and generally will be).  Thank you for the update.